library/src/main/java/org/xmtp/android/library/SigningKey.kt

```diff
@@ -50,7 +53,7 @@ fun SigningKey.createIdentity(
     val signatureClass = Signature.newBuilder().build()
     val signatureText = signatureClass.createIdentityText(key = slimKey.toByteArray())
     val digest = signatureClass.ethHash(message = signatureText)
-    val signature = runBlocking { sign(signatureText) } ?: throw XMTPException("Illegal signature")
```

Comment:
I think we can use the same `suspend` function for `sign` and `signLegacy` now

Reply:
This was just a rename so to not break existing stuff I'm just going to leave it for now. Probably a larger refactor to tighten up all the sign stuff.